index.html

```diff
@@ -175,36 +238,42 @@ <h1>About me</h1>
 
             <hr id="about-hr"/>
 
-            <ul id="skills-block">
-                <div class="skill-title-block">
-                    <div id="languages-logo" class="about-logo"></div>
-                    <h2 class="skill-title">Languages</h2>
-                </div>
-                <li>JavaScript</li>
-                <li>Ruby</li>
-                <li>HTML</li>
-                <li>CSS</li>
+            <div id="skills-block">
+                <ul>
+                    <div class="skill-title-block">
+                        <div id="languages-logo" class="about-logo"></div>
+                        <h2 class="skill-title">Languages</h2>
+                    </div>
+                    <li>JavaScript</li>
+                    <li>Ruby</li>
+                    <li>HTML</li>
+                    <li>CSS</li>
+                </ul>
 
-                <div class="skill-title-block">
-                    <div id="frameworks-logo" class="about-logo"></div>
-                    <h2 class="skill-title">Frameworks</h2>
-                </div>
-                <li>Bootstrap</li>
-                <li>Ruby on Rails</li>
-                <li>RSPec</li>
-                <li>CapyBara</li>
-                <li>Selenium</li>
+                <ul>
+                    <div class="skill-title-block">
+                        <div id="frameworks-logo" class="about-logo"></div>
+                        <h2 class="skill-title">Frameworks</h2>
+                    </div>
+                    <li>Bootstrap</li>
+                    <li>Ruby on Rails</li>
+                    <li>RSPec</li>
+                    <li>CapyBara</li>
+                    <li>Selenium</li>
+                </ul>
                     
-                <div class="skill-title-block">
-                    <div id="skills-logo" class="about-logo"></div>
-                    <h2 class="skill-title">Skills</h2>
-                </div>
-                <li>Codekit</li>
-                <li>Github</li>
-                <li>Codepen</li>
-                <li>Gitlab</li>
-                <li>Terminal</li>      
-            </ul>
+                <ul>
+                    <div class="skill-title-block">
+                        <div id="skills-logo" class="about-logo"></div>
+                        <h2 class="skill-title">Skills</h2>
+                    </div>
+                    <li>Codekit</li>
+                    <li>Github</li>
+                    <li>Codepen</li>
+                    <li>Gitlab</li>
+                    <li>Terminal</li> 
+                </ul>    
+            </div>
         </section>
         <!-- About Me section end -->
```

Comment:
- [ ] Please make sure that your contact form is similar to the  Figma template, especially your form textarea  it is important to follow the Figma template  :smile: 
#### Your design
![Screenshot from 2022-09-29 21-33-43](https://user-images.githubusercontent.com/6615394/193147336-719f9181-4b78-49b8-895e-2cc78ce77b53.jpg)

#### Figma design

![Screenshot from 2022-09-29 21-41-52](https://user-images.githubusercontent.com/6615394/193147460-ad80e3eb-da87-4cb2-a22e-8aeab6150a4a.png)